lib/keystore/encrypt.go

```diff
@@ -137,8 +137,17 @@ func EncryptAndWriteToFile(file *os.File, pk crypto.PrivateKey, password []byte)
 		return err
 	}
 
-	_, err = file.Write(append(data, byte('\n')))
-	return err
+	_, err = file.Write(data)
+	if err != nil {
+		return err
+	}
+
+	_, err = file.Write([]byte{'\n'})
+	if err != nil {
+		return err
+	}
```

Comment:
I prefer the way that uses only one `Write`.
```suggestion
	_, err = file.Write(append(data, byte('\n')))
	if err != nil {
	    return err
	}
```

Reply:
So you're happy with the changes right? Or unhappy? Just making sure :smile: